src/ModelSelectionUtils.hpp

```diff
@@ -0,0 +1,21 @@
+#ifndef MODELSELECTIONUTILS_H
+#define MODELSELECTIONUTILS_H
+
+#include <iostream>
+#include <map>
+#include <vector>
+
+#include "ModelSelectionUtils.hpp"
```

Comment:
why are you doing #include of this file into itself?? please delete

Reply:
Oh! I must have done that on accident. Will delete.